docs/collecting-data/collecting-from-own-applications/golang-tracker/emitters/index.md

```diff
@@ -9,21 +9,20 @@ Tracker instances must be initialized with an emitter. This section will go into
 The simplest Emitter setup requires only the collector URI to be passed to it:
 
 ```go
-emitter := sp.InitEmitter(RequireCollectorUri("com.acme"))
+emitter := sp.InitEmitter(RequireCollectorUri("com.acme"), *sp.RequireStorage(storagememory.Init()))
```

Comment:
Here the `*` is outside and below it’s inside (applied to `storagememory.Init()`). Which one is it?

Reply:
Should be inside - this one is a mistake.  Good catch!